addons/tmi/services/heat.gd

```diff
@@ -0,0 +1,75 @@
+extends TmiService
+class_name TmiHeatService
```

Comment:
please remove everything Heat related from the PR as it's unrelated

Reply:
Yeah sorry - The pull request was actually _for_ the Heat additions, but when Breadbotnik reported having issues getting tmi working, I made the other changes and Github just piled them onto the same pull request.